### PR TITLE
Standby cli test

### PR DESCRIPTION
**Pull request by mmichal10**

### CODE REVIEW

File: test/functional/tests/cli/test_cli_standby.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#
+# Copyright(c) 2019-2021 Intel Corporation
```
Comment (pdebski21) [inline test/functional/tests/cli/test_cli_standby.py:2]: update license to 2022

File: test/functional/tests/cli/test_cli_standby.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    for name, value in init_disallowed_params.items():
+        with TestRun.step(f'Try to init standby instance with disallowed "{name}" param'):
+            tested_param = f"{name} {value}"
+            tested_cmd = f"{valid_cmd} {tested_param}"
+            output = TestRun.executor.run(tested_cmd)
+            if output.exit_code == 0:
+                TestRun.LOGGER.error(
+                    f'"{tested_cmd}" command succeeded despite disallowed "{name}" paramter!'
+                )
+            if not check_stderr_msg(output, disallowed_param) or name not in output.stderr:
```
Comment (pdebski21) [inline test/functional/tests/cli/test_cli_standby.py:77]: Do we need to check that precisely if disallowed param is in error message?

Comment (mmichal10) [reply test/functional/tests/cli/test_cli_standby.py]: Good point

**Review by pdebski21 — DISMISSED** [review]

**Comment by mmichal10** [comment]
Retest this please

**Review by kmajzero — APPROVED** [review]

**Review by pdebski21 — DISMISSED** [review]

**Comment by mmichal10** [comment]
retest this please

**Review by kmajzero — APPROVED** [review]

**Review by pdebski21 — APPROVED** [review]

**Pull request merged**